### PR TITLE
Do not use customer as user and move fetching of events

### PR DESCRIPTION
**Pull request by HALLERPierre**
[Trello Card #455](https://trello.com/c/7F64D3tJ/455-enhance-audit-log-on-react-admin-demo) 
## Todo

- [x] Do not use customer as admin user
- [x] Fetch events in Timeline component instead of Dashboard
- [x] Add maxWidth to right col in desktop

## Screenshot(s)

image

### CODE REVIEW

**Review by djhi — APPROVED** [review]

File: src/authProvider.ts [hunk 1]
```diff
@@ -14,7 +14,14 @@ const authProvider: AuthProvider = {
     checkAuth: () =>
         localStorage.getItem('username') ? Promise.resolve() : Promise.reject(),
     getPermissions: () => Promise.reject('Unknown method'),
-    getIdentity: () => Promise.resolve({ id: 'demo', fullName: 'Demo' }),
+    getIdentity: () => Promise.resolve(demoUser),
+};
+
+export const demoUser = {
+    id: 'demo',
```
Comment (fzaninotto) [inline src/authProvider.ts:21]: The current user should be one of the admins that you have added in generateFakeEvents.

File: src/fakeServer/generateFakeEvents.ts [hunk 2]
```diff
@@ -67,15 +64,37 @@ const changeOrderStatusEvent = (
...
 
+const getRandomId = length => {
+    return Math.floor(Math.random() * length);
+};
+
+const admins = [
+    demoUser,
+    {
+        id: 0,
+        fullName: 'Iris Bailey',
```
Comment (fzaninotto) [inline src/fakeServer/generateFakeEvents.ts:75]: Could you add an avatar to each of the admins? The demo looks a bit sad without it.

File: src/dashboard/Timeline.tsx [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
-const MyTimeline = ({ records }: { records?: EventRecord[] }): ReactElement => {
+const MyTimeline = (): ReactElement => {
     const translate = useTranslate();
+    const { data: events, loaded } = useGetList<EventRecord>(
+        'events',
+        { perPage: 100, page: 1 },
+        { field: 'date', order: 'DESC' },
+        {}
+    );
+    const records = Object.values(events);
```
Comment (fzaninotto) [inline src/dashboard/Timeline.tsx:14]: you can't be sure of the order in which the values come. To get the records, you should use `ids.map(id => events[id])`, where `ids` comes from the result of `useGetList`

File: src/fakeServer/generateFakeEvents.ts (hunk 2 above)
Comment (HALLERPierre) [reply src/fakeServer/generateFakeEvents.ts:75]: uifaces.com

**Pull request merged**